jenkins/bootstrap.py

```diff
@@ -735,11 +737,16 @@ def job_args(args):
     return [os.path.expandvars(a) for a in args]
 
 
-def job_script(job, use_json):
+def job_script(job, use_json, jobs_dir):
     """Return path to script for job."""
+    jobs_dir_resolver = lambda f: test_infra('jobs/%s' % f)
+
+    if jobs_dir:
+        jobs_dir_resolver = lambda f: os.path.join(jobs_dir, f)
+
     if not use_json:
```

Comment:
hummm good to see the discussion here since I'm about to clean up all *.sh support. Can I see your  job config file within your prow?